### PR TITLE
Play 2.7 and Scala 2.12.8

### PR DESCRIPTION
**Pull request by balagez**
https://www.playframework.com/documentation/2.7.x/Migration27

### CODE REVIEW

**Review by ghost — APPROVED** [review]

**Comment by balagez** [comment]
Made a few changes since review:
* sbt 1.2
* drop support for Scala 2.10 (Play dropped support so we can no longer compile macrame-play)
* Fixed a few compiler warnings
* Bumped version numbers so that I can release everything
* Chris -> Claire

File: macrame/src/test/scala/macrame/RefClassTest.scala [hunk 1]
```diff
@@ -30,7 +30,7 @@ abstract class RefClassTest {
 }
 object RefClassTest {
    def apply[X[_] <: Ref[_], Y[_] <: Ref[_]](
-      id : Id[Post],
+      id :     Id[Post],
```
Comment (ghost) [inline macrame/src/test/scala/macrame/RefClassTest.scala:33]: That doesn't sit well with our code style. This is a forked repo, so, probably no reason it should, but still — are you sure aligning stuff like that makes sense?

**Review by ghost — APPROVED** [review]

File: macrame/src/test/scala/macrame/RefClassTest.scala (hunk 1 above)
Comment (balagez) [reply macrame/src/test/scala/macrame/RefClassTest.scala:33]: Scalariform upgrade did this. I'll look into whether I can do anything about it.

Comment (ghost) [reply macrame/src/test/scala/macrame/RefClassTest.scala:33]: Ouch. OK, good to know.

Comment (balagez) [reply macrame/src/test/scala/macrame/RefClassTest.scala:33]: Well actually this setting is enabled in this project:
```
.setPreference(AlignParameters, true)
```
So the thing is that Scalariform did not correctly align parameters before, but now it does. So this formatting change actually sits well with this repo.

**Pull request merged**